doc/cephadm/host-management.rst

```diff
@@ -94,6 +94,18 @@ Once all daemons are removed you can remove the host with the following:
 
   ceph orch host rm <host>
 
+Offline host removal
+--------------------
+
+If a host is offline and can not be recovered it can still be removed from the cluster with the following:
+
+.. prompt:: bash #
+
+  ceph orch host rm <host> --offline --force
+
+This can potentially cause data loss as osds will be forcefully purged from the cluster.
```

Comment:
could you add a sentence that we're calling `osd purge-actual` for each osd on that host?